.prettierrc

```diff
@@ -3,5 +3,6 @@
   "printWidth": 120,
   "trailingComma": "all",
   "singleQuote": true,
-  "semi": false
+  "semi": false,
+  "endOfLine": "auto"
```

Comment:
This removes the `[eslint] Delete 'cr' [prettier/prettier]` error when running `eslint`.